devices.js

```diff
@@ -9936,7 +9936,7 @@ const devices = [
         vendor: 'eWeLink',
```

Comment:
Question, are these the same as the sonoff devices? (https://sonoff.tech/product/smart-home-security/snzb-03). If so I think we should mark these all as Sonoff (I can do this after this PR).

Reply:
yes. modelID is the same for all devices

Reply:
but outside they look some different :)

Reply:
Would it make sense to mark them as Sonoff and add Ewelink as the whitelabel? (as sonoff is more common?)